src/doc.c

```diff
@@ -2325,6 +2327,41 @@ void highlightText(Scope *sc, Dsymbol *s, OutBuffer *buf, size_t offset)
                 }
                 break;
 
+            case '`':
+                if (inBacktick)
+                {
+                    inBacktick = 0;
+                    inCode = 0;
+
+                    OutBuffer codebuf;
+
+                    codebuf.write(buf->data + iCodeStart + 1, i - (iCodeStart + 1));
+                    codebuf.writeByte(0);
+
+                    highlightCode2(sc, s, &codebuf, 0);
+
+                    buf->remove(iCodeStart, i - iCodeStart + 1); // also trimming off the current `
+                    i = buf->insert(iCodeStart, codebuf.data, codebuf.offset);
+                    i = buf->insert(i, ")", 1);
+
+                    break;
+                }
+
+                if (inCode)
+                    break;
+
+                inCode = 1;
+                inBacktick = 1;
+
+                static const char pre[] = "$(D_INLINECODE ";
```

Comment:
Oh, I now understand your point. But then we have a larger problem. Unlike code snippets, inline code should not be highlighted. If it were, it'd just make the text too jarring to read. In TDPL all snippets use distinctive font for keywords in snipped but not in running text; experiments to do otherwise made the text look too loaded.

So I say let's convert `hello world` to `$(D_BACKQUOTED hello world)`. That's the right solution. Thanks.